### PR TITLE
19442 - testing accessibility: after expanding the form (add new SI on owner change)

### PR DESCRIPTION
**Pull request by hfekete**
*Issue:19442 *https://github.com/bcgov/entity/issues/19442 

*Description of changes:*
- testing accessibility: after expanding the form (add new SI on owner change)

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namex license (Apache 2.0).

### CODE REVIEW

File: btr-web/btr-main-app/components/individual-person/tax-info/TaxNumber.vue [hunk 1]
```diff
@@ -6,6 +6,7 @@
         v-model="selectedButton"
         :value="HAS_TAX_NUMBER"
         class="mt-3"
+        :aria-label="$t('placeholders.taxNumber')"
```
Comment (kialj876) [inline btr-web/btr-main-app/components/individual-person/tax-info/TaxNumber.vue:9]: ahhh nice I didn't think about having these internationalized

File: btr-web/btr-main-app/cypress/e2e/accessibility/ownerChange/expandingTheForm.cy.ts [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+  it('checks page passes accessibility', () => {
+    // For now, this is an example of passing checks
+    cy.checkA11y('[data-cy=page-header]')
+    cy.checkA11y('[data-cy=page-info-text]')
+    cy.checkA11y('[data-cy=add-new-btn]')
+
+    // NB: these tests will be uncommented when their corresponding tickets are worked on
+
+    // Expanding the form - pre filled out 19442
```
Comment (kialj876) [inline btr-web/btr-main-app/cypress/e2e/accessibility/ownerChange/expandingTheForm.cy.ts:22]: I think it makes more sense to keep this test inside the file it was in previously and inside its own 'it'. That's also what we've done for the other tests

Comment (hfekete) [reply btr-web/btr-main-app/cypress/e2e/accessibility/ownerChange/expandingTheForm.cy.ts]: Updated

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-btr-dev--pr-123-rvjh5hfw.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-btr-dev--pr-123-rvjh5hfw.web.app

**Review by kialj876 — APPROVED** [review]
thanks!

**Pull request merged**